spec/lib/danger/plugin_support/gems_resolver_spec.rb

```diff
@@ -50,7 +50,9 @@ def fake_bundle_install_path_vendor_gems_in(spec_root)
           fake_bundle_install_path_vendor_gems_in(spec_root)
         end
 
-        result = described_class.new(gem_names).call
+        resolver = described_class.new(gem_names)
+        resolver.call
+        result = resolver.send(:all_gems_metadata)
```

Comment:
also a bit inelegant, but we can't call the original block here, I don't feel too bad calling the method after